p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -1427,6 +1428,28 @@ Mpls_h[10] mpls;
 ~ End P4Example
 introduce a header stack called ```mpls``` containing 10 entries, each of type ```Mpls_h```.
 
+### Header Unions {#sec-header-unions}
```

Comment:
There is another table that @mbudiu-vmw may have been thinking of, that it seems you may not have modified @jnfoster.  I have a separate PR for that one now: https://github.com/p4lang/p4-spec/pull/190